application/controllers/bestitamazoncron.php

```diff
@@ -17,6 +19,23 @@ class bestitAmazonCron extends oxUBase
      */
     protected $_oContainer = null;
 
+    /**
+     * The logger
+     *
+     * @var LoggerInterface
+     */
+    protected $_oLogger;
+
+    /**
+     * bestitAmazonCron constructor.
+     */
+    public function __construct()
+    {
+        $this->_oLogger = $this->_getContainer()->getLogger();
```

Comment:
Isn't that dangerous to call the container at constructor?

Reply:
The container itsellf will be instatiated with oxNew. So it should be a problem if this will be executed in the constructor.